GCSViews/ConfigurationView/ConfigRawParams.cs

```diff
@@ -266,6 +263,49 @@ private void BUT_writePIDS_Click(object sender, EventArgs e)
             int error = 0;
             bool reboot = false;
 
+            if (temp.Count > 0 && temp.Count <= 20)
+            {
+                // List to track successfully saved parameters
+                List<string> savedParams = new List<string>();
+
+                foreach (string value in temp)
+                {
+                    if (MainV2.comPort.BaseStream == null || !MainV2.comPort.BaseStream.IsOpen)
+                    {
+                        CustomMessageBox.Show("You are not connected", Strings.ERROR);
+                        return;
+                    }
+
+                    // Get the previous value of the param to display in 'param change info'
+                    // (a better way would be to get the value somewhere from inside the code, insted of recieving it over mavlink)
+                    string previousValue = MainV2.comPort.MAV.param[value].ToString();
+                    // new value of param
+                    double newValue = (double)_changes[value];
+
+                    // Add the parameter, previous and new values to the list for 'param change info'
+                    // remember, the 'value' here is key of param, while prev and new are actual values of param
+                    savedParams.Add($"{savedParams.Count + 1})  {value}  : {previousValue}  ->  {newValue}");

```

Comment:
I'm not a fan of the numbered list. Would prefer this
```suggestion
                    savedParams.Add($"{value}: {previousValue}  ->  {newValue}");
```

Reply:
removed numbered list in [commit id 22c75ed](https://github.com/ArduPilot/MissionPlanner/pull/3412/commits/22c75ed012c5702bd9eb70df164d8166320fa3e3)